### PR TITLE
Ci: Automatically sync all translations from transifex

### PR DESCRIPTION
**Pull request by TheOneRing**

### CODE REVIEW

File: translations/.tx/config [hunk 1]
```diff
@@ -2,52 +2,15 @@
...
-
-# choose a special language as more generic default
-trans.cs_CZ = client_cs.ts
-trans.et_EE = client_et.ts
-trans.fi_FI = client_fi.ts
-trans.ja_JP = client_ja.ts
-trans.hu_HU = client_hu.ts
-trans.sk_SK = client_sk.ts
-trans.th_TH = client_th.ts
+lang_map = de_DE: de, pt_PT: pt, cs_CZ: cs, et_EE: et, fi_FI: fi, ja_JP: ja, hu_HU: hu, sk_SK: client_sk, th_TH: th
```
Comment (TheOneRing) [inline translations/.tx/config:11]: @fmoc is this mapping still needed/up to date with your recent changes and if so maybe we can still get rid of it?

**Comment by TheOneRing** [comment]
Translations that will be added after the merge:
```
        translations/client_ar.ts
        translations/client_bg_BG.ts
        translations/client_de_CH.ts
        translations/client_en_GB.ts
        translations/client_he.ts
        translations/client_is.ts
        translations/client_ko.ts
        translations/client_sq.ts
        translations/client_vi.ts

**Comment by TheOneRing** [comment]
As I also got 
```
C:\CraftRoot\build\_\30b36579\owncloud-client [work/translations +10 ~21 -0 !]> ls .\translations\.tx\owncloud.client-desktop\

    Directory: C:\CraftRoot\build\_\30b36579\owncloud-client\translations\.tx\owncloud.client-desktop

Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
-a---          31/05/2021    15:42            919 af_ZA_translation
-a---          31/05/2021    15:42            918 ar_translation
-a---          31/05/2021    15:42           1002 bg_BG_translation
-a---          31/05/2021    15:42            932 ca_translation
-a---          31/05/2021    15:42            941 cs_CZ_translation
-a---          31/05/2021    15:42            910 da_translation
-a---          31/05/2021    15:42            925 de_AT_translation
-a---          31/05/2021    15:42            884 de_CH_translation
-a---          31/05/2021    15:42            923 de_DE_translation
-a---          31/05/2021    15:42            911 de_translation
-a---          31/05/2021    15:42           1045 el_translation
-a---          31/05/2021    15:42            902 en_GB_translation
-a---          31/05/2021    15:42            956 es_AR_translation
-a---          31/05/2021    15:42            958 es_MX_translation
-a---          31/05/2021    15:42            935 es_translation
-a---          31/05/2021    15:42            930 et_EE_translation
-a---          31/05/2021    15:42            922 eu_translation
-a---          31/05/2021    15:42            882 fa_translation
-a---          31/05/2021    15:42            923 fi_FI_translation
-a---          31/05/2021    15:42            905 fr_translation
-a---          31/05/2021    15:42            933 gl_translation
-a---          31/05/2021    15:42            937 he_translation
-a---          31/05/2021    15:42            926 hu_HU_translation
-a---          31/05/2021    15:42            932 ia_translation
-a---          31/05/2021    15:42            884 id_translation
-a---          31/05/2021    15:42            909 is_translation
-a---          31/05/2021    15:42            936 it_translation
-a---          31/05/2021    15:42            946 ja_JP_translation
-a---          31/05/2021    15:42            921 ko_translation
-a---          31/05/2021    15:42            891 lb_translation
-a---          31/05/2021    15:42           1004 lo_translation
-a---          31/05/2021    15:42            921 lt_LT_translation
-a---          31/05/2021    15:42            862 lv_translation
-a---          31/05/2021    15:42            995 mk_translation
-a---          31/05/2021    15:42            925 nb_NO_translation
-a---          31/05/2021    15:42            889 nl_translation
-a---          31/05/2021    15:42            945 nn_NO_translation
-a---          31/05/2021    15:42            905 oc_translation
-a---          31/05/2021    15:42            952 pl_translation
-a---          31/05/2021    15:42            942 pt_BR_translation
-a---          31/05/2021    15:42            930 pt_PT_translation
-a---          31/05/2021    15:42            916 ro_translation
-a---          31/05/2021    15:42            995 ru_translation
-a---          31/05/2021    15:42            937 sl_translation
-a---          31/05/2021    15:42            919 sq_translation
-a---          31/05/2021    15:42            885 sv_translation
-a---          31/05/2021    15:42           1106 th_TH_translation
-a---          31/05/2021    15:42            904 tr_translation
-a---          31/05/2021    15:42            984 uk_translation
-a---          31/05/2021    15:42            898 zh_CN_translation
-a---          31/05/2021    15:42            898 zh_TW_translation
```

I assume the desktop file translations where broken ...

**Comment by TheOneRing** [comment]
https://docs.transifex.com/formats/freedesktop doesn't really help

**Review by michaelstingl — APPROVED** [review]
❤️ it!!

**Review by hosy — APPROVED** [review]
LGTM

File: translations/.tx/config (hunk 1 above)
Comment (dschmidt) [reply translations/.tx/config:11]: is there no multiline notation of this mapping?

would be nicer to review diffs later on .. but okay anyway

**Review by dschmidt — APPROVED** [review]
if it works, 🚢

**Pull request merged**